### PR TITLE
Second read/review

### PR DESCRIPTION
**Pull request by alexkirsz**
Some comments similar to https://github.com/DioxusLabs/docsite/pull/1, some new comments.

### CODE REVIEW

File: posts/release.md [hunk 1]
```diff
@@ -82,54 +80,55 @@ To get started with Dioxus, check out any of the "Getting Started" guides for yo
...
 
-We do support some differences from React as features:
+Some features also consciously diverge from React idioms:
 
 - Automatic memoization (opt-out rather than opt-in)
 - No use_effect hook (yes, this is a feature)
-- Suspense is implemented as hooks, *not* deeply ingrained in Dioxus Core
-- Async code is *explicit* with a preference to *coroutines* instead
+- Suspense is implemented as hooks, _not_ deeply ingrained within Dioxus Core
+- Async code is _explicit_ with a preference for _coroutines_ instead
```
Comment (alexkirsz) [inline posts/release.md:88]: I think this is my markdown formatter doing its thing.

File: posts/release.md [hunk 2]
```diff
@@ -82,54 +80,55 @@ To get started with Dioxus, check out any of the "Getting Started" guides for yo
...
-In contrast, Dioxus is written in Rust - which is almost like "TypeScript on steroids".
+In contrast, Dioxus is written in Rust, which is almost like "TypeScript on steroids".
 
 By using Rust, we gain:
 
 - Static types for every library
 - Immutability by default
 - A simple and intuitive module system
-- Integrated documentation (go to source actually... goes to source instead of the `.d.ts` file)
+- Integrated documentation (go to source actually goes to source instead of the `.d.ts` file)
```
Comment (alexkirsz) [inline posts/release.md:112]: Thought this sounded a little bit snarky!

File: posts/release.md [hunk 3]
```diff
@@ -148,46 +147,52 @@ static Card: Component<CardProps> = |cx, props| {
...
-			<h2>{ props.title }</h2>
-			<p> { props.paragraph } </p>
-		</aside>
-	)
+  return (
+    <aside>
+      <h2>{props.title}</h2>
+      <p> {props.paragraph} </p>
+    </aside>
+  );
```
Comment (alexkirsz) [inline posts/release.md:163]: Also my formatter.

File: posts/release.md [hunk 4]
```diff
@@ -148,46 +147,52 @@ static Card: Component<CardProps> = |cx, props| {
...
 
+### Building a new project is simple!
 
-### Building a new Project is simple!
+To start a new project, all you need is Cargo, which comes with Rust. For a simple desktop app, all we'll need is the `dioxus` crate with the appropriate `desktop` feature. We start by initializing a new binary crate:
+
+```shell
+cargo init dioxus_example
+cd dioxus_example
+```
```
Comment (alexkirsz) [inline posts/release.md:180]: Thought not everyone reading the blog post would be familiar with Cargo.

**Review by alexkirsz — COMMENTED** [review]
Added comments in sources within <<<>>> to avoid forgetting later :)

**Review by jkelleyrtp — APPROVED** [review]
Thank you!

**Pull request merged**